test/models/patient_test.rb

```diff
@@ -646,6 +646,41 @@ class PatientTest < ActiveSupport::TestCase
     end
   end
 
+  describe 'all_versions' do
+    before do
+      # For some reason bullet doesn't play nicely with these unit tests,
+      # but does with the system tests. Given the choice I prefer the system
+      # tests, so we assume this is a false negative and turn off bullet for these.
```

Comment:
Should there be system tests in this PR?

Reply:
nah, by this I meant that system tests would trigger any problems in bullet (though if there's a behavior you want to see locked in, definitely _can_ add system tests)